### PR TITLE
[ZEPPELIN-4760] Add description and homepage information on github repository

### PR DESCRIPTION
**Pull request by Leemoonsoo**
### What is this PR for?

I'd like to discuss and get ideas on how we can improve Zeppelin GitHub repository description, which looks like
image

### What type of PR is it?
Improvement

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-4760

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# https://cwiki.apache.org/confluence/display/INFRA/.asf.yaml+features+for+git+repositories
+
+github:
+  description: "Web-based notebook that enables data-driven, interactive data analytics and collaborative documents with SQL, Scala and more."
```
Comment (Leemoonsoo) [inline .asf.yaml:21]: I copied this description from the website https://zeppelin.apache.org.
Does someone have a better idea?

File: .asf.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  homepage: https://zeppelin.apache.org/
+  labels:
+    - scala
+    - java
+    - big-data
+    - zeppelin
+    - javascript
```
Comment (Leemoonsoo) [inline .asf.yaml:28]: I just copied the current labels on the repository here. But feel like they need to be improved. Any suggestions?

Comment (alexott) [reply .asf.yaml:28]: I would add `spark`, `flink`, `database`, `nosql` - the first two are very popular.

Comment (Leemoonsoo) [reply .asf.yaml:32]: Added them on the list. Thanks for the suggestion.

I'll merge this to master, while it can be updated anytime via pull request.

**Pull request closed without merging**